chrome/content/zotero/elements/styleConfigurator.js

```diff
@@ -33,19 +31,26 @@
 				xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
 			>
 				<div>
-					<xul:richlistbox id="style-list" tabindex="0" />
+					<xul:richlistbox id="style-list" class="theme-listbox" tabindex="0" />
 				</div>
 			</div>
 		`);
 
 		set value(val) {
-			this.querySelector('#style-list').value = val;
+			if (!this.values.includes(val)) return;
+			let styleList = this.querySelector('#style-list');
+			styleList.value = val;
+			styleList.selectedItem.scrollIntoView();
```

Comment:
We should only scroll if it's not already visible. Compare to Z6:

![Screenshot 2024-06-27 at 1 58 40 AM](https://github.com/zotero/zotero/assets/783082/8b24bf19-9cac-4ae4-96a8-a2c03f539254)

It would also be nice to scroll the selected style a little bit further into view instead of being right against the top:

![Screenshot 2024-06-27 at 1 55 54 AM](https://github.com/zotero/zotero/assets/783082/5272fb9c-83ac-4250-b0de-1bd3b3bb7d51)